### ISSUE TITLE
Inconsistency with regard to ARIA attributes

### ISSUE THREAD

**Issue by torcellite** [PolymerElements/paper-toggle-button#21]
The test cases used `aria-checked` instead of `aria-pressed` which is the prescribed `host:attribute`, which wasn't much of an issue to change. However, when I changed it to `aria-pressed`, the attribute is toggled regardless of the disabled state. This fails the last test case.

```
test('disabled button cannot be clicked', function() {
      b1.disabled = true; // this has no effect
      b1.checked = true;
     // aria-pressed is now true
      b1.addEventListener('click', function() {
        // aria-pressed is false here and so the test fails
        assert.isTrue(b1.getAttribute('aria-pressed') == 'true');
        assert.isTrue(b1.checked);
      });
      MockInteractions.tap(b1);
      // aria-pressed is now false because of the tap
    });
```

![screen shot 2015-06-20 at 1 18 29 pm](https://cloud.githubusercontent.com/assets/3049807/8266525/21a54c80-1750-11e5-90f9-10a4111585f8.png)

So I fiddled with the element and replaced `checked` with  `pressed`. This caused an issue in `iron-behaviors`,  the `_setPressed` function was missing in `iron-button-state.html`. I added the function and did not receive any errors in the console, but the `_ontrack` listener was never called. I didn't know how to proceed any further, hence, the issue.

**Comment by notwaldorf** [comment]
I think this has been fixed (all the tests use `aria-pressed`)
